apps/ledger-live-desktop/src/renderer/reducers/settings.ts

```diff
@@ -121,7 +121,7 @@ export const getInitialLocale = () => {
 };
 
 const INITIAL_STATE: SettingsState = {
-  hasCompletedOnboarding: false,
+  hasCompletedOnboarding: true,
```

Comment:
probably a mistake

Reply:
Maybe relevant to bypass onboarding I assume, which is Nano-specific.
OK for a custom build, but won't likely be mergeable on Live codebase.

Reply:
Yep that's correct: it's a bypass that we will remove before asking for a merge into LL

Reply:
Removed 👍